CMakeLists.txt

```diff
@@ -141,38 +143,30 @@ if (USE_ROOT)
 endif()
 
 #------------------------------------------------------------------------------
-#Build the Core library
+#Include and build the Resources that everybody should have access to.
+include_directories(Resources/include)
+add_subdirectory(Resources)
+
+#Build the PaassCoreLibrary.
 include_directories(Core/include)
 add_subdirectory(Core)
 
-#Build the scan library
-add_subdirectory(Scan)
-
-#Building polling tools
-if (BUILD_SUITE)
-	#Adds the install prefix for referencing in the source code
-	add_definitions(-D INSTALL_PREFIX="\\"${CMAKE_INSTALL_PREFIX}\\"")
+#Next we are going to build any of the ThirdParty software that we've included.
+include_directories(ThirdParty/include)
+#We'll go straight to the source directory since for now we don't need any
+#fancy logic in the base directory.
+add_subdirectory(ThirdParty)
 
-	#Build the pixie interface
-	include_directories(Interface/include)
-	add_subdirectory(Interface/source)
 
-	#Build the MCA objects
-	include_directories(MCA/include)
-	add_subdirectory(MCA)
 
-	#Build the setup tools
-	if (BUILD_SETUP) 
-		include_directories(Setup/include)
-		add_subdirectory(Setup)
-	endif()
-
-	#Build poll
-	add_subdirectory(Poll)
+#Build Acquisition software, we don't have this broken into as fine of a
+#granularity as the Analysis software, so we just wrap the whole thing in an if.
+if (BUILD_SUITE)
```

Comment:
You know, I had started to do that, but changed my mind midway through. I stopped because I didn't want to figure out what/how to change things for the Scan part.  **I'll change it. It makes sense.**